src/plugins/intel_npu/src/plugin/src/compiled_model.cpp

```diff
@@ -139,6 +150,12 @@ void CompiledModel::export_model(std::ostream& stream) const {
     std::stringstream str;
     str << "Blob size: " << blob.size() << ", hash: " << std::hex << hash(blob);
     _logger.info(str.str().c_str());
+    // if graphHandle is not null, release blob here to reduce peak mem
```

Comment:
// If graphHandle is not a nullptr it means there is still an instance of the blob maintained inside the driver and we can release the copy of the blob here to reduce memory consumption.